treeherder/model/models.py

```diff
@@ -246,7 +249,15 @@ def serialize(self):
 
         attrs = model_to_dict(self, exclude=exclude_fields)
         # Serialize bug ID as the bugzilla number for compatibility reasons
+        attrs["internal_id"] = attrs["id"]
         attrs["id"] = attrs.pop("bugzilla_id")
+
+        attrs["occurrences"] = None
+        if attrs["id"] is None:
+            # Only fetch occurrences for internal issues. It causes one extra query
+            attrs["occurrences"] = BugJobMap.objects.filter(
+                created__gte=timezone.now() - self.INTERNAL_OCCURRENCES_WINDOW
```

Comment:
does this need to be filtered down on a specific id, not just time?

Reply:
This is the `Bugscache` serialization, not a lookup in the `BugJobMap` - no issue ID needed.